### PR TITLE
feat: translated navigations and page titles to nepali

### PR DESCRIPTION
**Pull request by archanakarki**
Translated: navigations and page titles of `src/locales/ne.json` to nepali

### CODE REVIEW

File: src/locales/ne.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      "emailLabel": "इ-मेल ठेगाना",
+      "whereDidYouHearLabel": "तपाईले चुनौतीको बिषयमा कहाँबाट सुन्नुभयो ?",
+      "submitButton": "पेश गर्नुहोस्",
+      "submitSuccessMessage": "धन्यवाद! हामी तपाईंलाई चाँडै सम्पर्क गर्नेछौं ।",
+      "participantsTitle": "फुल स्ट्याक चुनौतीमा सामेल भएका कम्पनीहरू",
+      "coOperationTitle": "सहकार्यमा",
+      "infoButton": "चुनौती बारे थप जानकारी"
+    },
+    "companiesPage": {
+      "storiesTitle": "हाम्रा साझेदारहरू(patners)बाट विकासकर्ताहरूको(Developers) कथाहरू"
```
Comment (archanakarki) [inline src/locales/ne.json:45]:
```suggestion
      "storiesTitle": "हाम्रा साझेदार/पार्टनरहरू बाट विकासकर्ता/डेभेलोपेरहरूको कथाहरू"
```

File: src/locales/ne.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    "aboutPage": {
+      "generalTitle": "सामान्य जानकारी"
+    },
+    "navigation": {
+      "skipToContent": "सामग्रीमा जानुहोस्",
+      "aboutCourse": "पाठ्यक्रम(कोर्ष)को बारे",
+      "courseContents": "पाठ्यक्रम/कोर्ष सामग्री",
+      "faq": "बारम्बार सोधिने प्रश्नहरू",
+      "partners": "साझेदारहरू(patners)",
```
Comment (archanakarki) [inline src/locales/ne.json:20]:
```suggestion
      "partners": "साझेदार/पार्टनरहरू",
```

File: src/locales/ne.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      "emailLabel": "इ-मेल ठेगाना",
+      "whereDidYouHearLabel": "तपाईले चुनौतीको बिषयमा कहाँबाट सुन्नुभयो ?",
+      "submitButton": "पेश गर्नुहोस्",
+      "submitSuccessMessage": "धन्यवाद! हामी तपाईंलाई चाँडै सम्पर्क गर्नेछौं ।",
+      "participantsTitle": "फुल स्ट्याक चुनौतीमा सामेल भएका कम्पनीहरू",
+      "coOperationTitle": "सहकार्यमा",
+      "infoButton": "चुनौती बारे थप जानकारी"
+    },
+    "companiesPage": {
+      "storiesTitle": "हाम्रा साझेदार/पार्टनरहरू बाट विकासकर्ता/डेभेलोपेरहरूको कथाहरू"
```
Comment (archanakarki) [inline src/locales/ne.json:45]: fix typo:
```suggestion
      "storiesTitle": "हाम्रा साझेदार/पार्टनरहरू बाट विकासकर्ता/डेभेलोपरहरूको कथाहरू"
```

**Comment by mluukkai** [comment]
no progress here so will not merge. a translation would of course be most welcome ❤️

**Pull request closed without merging**